### PR TITLE
Post Publication end

### PR DESCRIPTION
**Pull request by BartlomiejSwit**

### CODE REVIEW

**Review by Shreemy — APPROVED** [review]
In my opinion, it is not that it is good or that it is not good. If I had to say what I value most in life, I would say people. Ekhm... People who gave me a helping hand when I was not coping, when I was alone. And interestingly enough, it's the chance encounters that influence our lives. The point is that when one adheres to certain values, even seemingly universal ones, there are times when one doesn't find the understanding, so to speak, that helps us grow. I was lucky, so to speak, because I found it. And I thank life. I thank it, life is singing, life is dancing, life is love. Many people ask me the same thing, but how do you do it, where do you get this joy? And I answer that it's simple, it's the love of life, it's what makes me build machines today, for example, and tomorrow.... who knows, why not, I will give myself to social work and will ot, for example, plant.... ie. carrots.

File: Library_WebServer/Controllers/PublicationsController.cs [hunk 1]
```diff
@@ -1,5 +1,8 @@
+using Library_WebServer.Database;
 using Library_WebServer.Models;
 using Microsoft.AspNetCore.Mvc;
+using System.Xml.Linq;
```
Comment (Zdunki) [inline Library_WebServer/Controllers/PublicationsController.cs:4]: Why this library added? We're not working with xmls.

File: Library_WebServer/Database/LibraryDbContext.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ namespace Library_WebServer.Database
 {
     public class LibraryDbContext : DbContext
     {
-        private const string CONNECTION_STRING = "Server=localhost;Port=5432;Database=library;User Id=pgadmin;Password=pgpass;";
+        private const string CONNECTION_STRING = "Server=localhost;Port=5432;Database=library;User Id=admin;Password=admin;";
```
Comment (Zdunki) [inline Library_WebServer/Database/LibraryDbContext.cs:9]: I don't mind it in this case, and we discussed it, but just as a reminder it's good to double check for changes like that and remove them before a PR.

File: Library_WebServer/Program.cs [hunk 3]
```diff
@@ -10,7 +10,7 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 builder.Services.AddDbContext<LibraryDbContext>(
-    options => options.UseNpgsql("Server=localhost;Port=5432;Database=library;User Id=pgadmin;Password=pgpass;"));
+    options => options.UseNpgsql("Server=localhost;Port=5432;Database=library;User Id=admin;Password=admin;"));
```
Comment (Zdunki) [inline Library_WebServer/Program.cs:13]: Same here

**Pull request merged**